code/modules/mining/ores_coins.dm

```diff
@@ -23,9 +23,13 @@
 	..()
 
 /obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/show_message = TRUE
+	for(var/obj/item/weapon/ore/O in loc)

```

Comment:
@lzimann it won't be so bad, I told him to put a ```break``` in so the moment it finds an ore it stops looping, the only time it'll be an issue is when there's a bunch of NON ores on the same turf as an ore, which atleast for mining will be a fairly rare occurrence.